src/Symfony/Component/Validator/Constraints/Ip.php

```diff
@@ -46,6 +47,16 @@ class Ip extends Constraint
     public const V6_ONLY_PUBLIC = '6_public';
     public const ALL_ONLY_PUBLIC = 'all_public';
 
+    // adds inverse FILTER_FLAG_NO_PRIV_RANGE
+    public const V4_ONLY_PRIV = '4_priv';
```

Comment:
PRIV is an abbreviation for "PRIVATE" ? If yes I would suffix the constants with PRIVATE instead of PRIV

Same for the other constants.

Excepts this is commonly used like this, what I don't know

Reply:
He reused the same syntax as the existing `*_NO_PRIV`